pysatMissions/instruments/pysat_ephem.py

```diff
@@ -186,31 +186,29 @@ def load(fnames, tag=None, inst_id=None, obs_long=0., obs_lat=0., obs_alt=0.,
 # create metadata corresponding to variables in load routine just above
 # made once here rather than regenerate every load call
 meta = pysat.Meta()
-meta['Epoch'] = {'units': 'Milliseconds since 1970-1-1',
-                 'Bin_Location': 0.5,
-                 'notes': 'UTC time at middle of geophysical measurement.',
-                 'desc': 'UTC seconds',
-                 'long_name': 'Time index in milliseconds'}
-meta['glong'] = {'units': 'degrees',
-                 'long_name': 'Geodetic longitude',
-                 'desc': 'WGS84 geodetic longitude'}
-meta['glat'] = {'units': 'degrees',
-                'long_name': 'Geodetic latitude',
-                'desc': 'WGS84 geodetic latitude'}
-meta['alt'] = {'units': 'km',
-               'long_name': 'Geodetic height',
-               'desc': "WGS84 height above Earth's surface"}
-meta['position_ecef_x'] = {'units': 'km',
-                           'desc': 'ECEF x co-ordinate of satellite'}
-meta['position_ecef_y'] = {'units': 'km',
-                           'desc': 'ECEF y co-ordinate of satellite'}
-meta['position_ecef_z'] = {'units': 'km',
-                           'desc': 'ECEF z co-ordinate of satellite'}
-meta['obs_sat_az_angle'] = {'units': 'degrees',
-                            'desc': 'Azimuth of satellite from ground station'}
-meta['obs_sat_el_angle'] = {'units': 'degrees',
-                            'desc': ' '.join(('Elevation of satellite from',
-                                              'ground station'))}
-meta['obs_sat_slant_range'] = {'units': 'km',
-                               'desc': ' '.join(('Distance of satellite from',
-                                                 'ground station'))}
+meta['Epoch'] = {meta.labels.units: 'Milliseconds since 1970-1-1',
+                 meta.labels.notes: ' '.join(('UTC time at middle of',
+                                              'geophysical measurement.')),
+                 meta.labels.desc: 'UTC seconds',
+                 meta.labels.name: 'Time index in milliseconds'}
+meta['glong'] = {meta.labels.units: 'degrees',
+                 meta.labels.desc: 'WGS84 geodetic longitude'}
+meta['glat'] = {meta.labels.units: 'degrees',
+                meta.labels.desc: 'WGS84 geodetic latitude'}
+meta['alt'] = {meta.labels.units: 'km',
+               meta.labels.desc: "WGS84 height above Earth's surface"}
+meta['position_ecef_x'] = {meta.labels.units: 'km',
+                           meta.labels.desc: 'ECEF x co-ordinate of satellite'}
+meta['position_ecef_y'] = {meta.labels.units: 'km',
+                           meta.labels.desc: 'ECEF y co-ordinate of satellite'}
+meta['position_ecef_z'] = {meta.labels.units: 'km',
+                           meta.labels.desc: 'ECEF z co-ordinate of satellite'}
+meta['obs_sat_az_angle'] = \
+    {meta.labels.units: 'degrees',
```

Comment:
```suggestion
meta['obs_sat_az_angle'] = {
    meta.labels.units: 'degrees',
```

Here and elsewhere.

Reply:
I like this.  Pushed an update.  Should I leave this unmerged until the main main branch is merged?

Reply:
Yes, I'm planning on re-triggering the tests on all these pulls after merging the meta one.